sources/packages/backend/apps/db-migrations/src/sql/Announcements/Create-announcements-table.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE IF NOT EXISTS sims.announcements (
+    id SERIAL PRIMARY KEY,
+    message_title VARCHAR(100) NOT NULL,
```

Comment:
Have we confirmed on the length of this to be 100?

Reply:
Please check with business to check on acceptable message title length.

Reply:
The message is defined to be a max length of 200. The title was not defined but should reasonably be shorter than the message itself. I've been told in past that business should not be bothered with exact db type/implementation details.

Reply:
For the message title, I do not see a huge risk, we can increase it to 250 if we want.
The message being 200 seems to be problematic. We can go for a higher number like 2500.

Reply:
The ticket requested 200 for the message. "Text - 200 char max, plain text".